### ISSUE TITLE
scheduler #replicas (32) and cluster size 

### ISSUE THREAD

**Issue by ferantivero** [mspnp/microservices-reference-implementation#54]
after getting everything up and running we see 32 desired replicas for scheduler and a 3-nodes-cluster (Standard_D2_v2).

**Comment by cfarre** [comment]
@ferantivero 
what is the issue? you do not see the stateful set count when you installed?
what is the work item?

**Comment by ferantivero** [comment]
**TL;DR**  creation of a relative small cluster in comparison to [the Scheduler desired state](https://github.com/mspnp/microservices-reference-implementation/blob/master/k8s/scheduler.yaml#L18). 

I'd rather reduce the number of replicas in [the Scheduler from 1 to 3](https://github.com/mspnp/microservices-reference-implementation/blob/master/k8s/scheduler.yaml#L18) than modify the deployment steps for the creation of a bigger cluster. 

Additionally, I think it's a way to get the foot in the door in the assessment of https://github.com/mspnp/microservices-reference-implementation/issues/33 from the Scheduler standpoint.

Therefore, the idea around this issue could be:

- [ ] figure out needed cluster size and/or reduce the requested resources by the Scheduler

- [ ] kind of measure Scheduler's Pod resource request/limit. Eventually, it'd be prescribed within the k8s configuration file contributing to address https://github.com/mspnp/microservices-reference-implementation/issues/33. 

Most probably, this information has already collected in the past.

**Comment by dragon119** [comment]
Reduce the number of EH partitions down to 3. Add comment about # of node, EH partitions. For higher throughput, we can increase this number but EH partition and scheduler stateful sets should match. Decide the # of replicas for other services. For demo, each service has 1 replica.

**Comment by MikeWasson** [comment]
The deployment defaults to 4 EH partitions now
